### PR TITLE
Design/region

### PR DESCRIPTION
**Pull request by jennyjyu**
Style regionCard and cityCard on RegionPage. Mostly width of different cards.

### CODE REVIEW

File: src/styles/card.ts [hunk 1]
```diff
@@ -59,11 +59,15 @@ export const cardImg = style({
...
 export const secondaryCard = style({
+    maxWidth: '400px',
     backgroundColor: 'rgb(236,236,236,1)',
     filter: 'drop-shadow(0px 4px 4px rgba(0, 0, 0, 0.25))',
     borderRadius: '6px',
     textAlign: 'center',
     padding: '5px',
+    marginTop: '5px',
+    marginLeft: 'auto',
+    marginRight: 'auto',
```
Comment (eirsture) [inline src/styles/card.ts:70]: Could you rewrite on the form: margin: '5px auto 0'?

File: src/components/RegionCard/styles.ts [hunk 2]
```diff
@@ -4,20 +4,19 @@ import {card} from '../../styles/card'
 
 export const regionCard = classes(
```
Comment (eirsture) [inline src/components/RegionCard/styles.ts:5]: Could you add: "cursor: pointer", so it shows as clickable on desktop?

**Review by eirsture — CHANGES_REQUESTED** [review]

File: src/components/RegionCard/styles.ts (hunk 2 above)
Comment (jennyjyu) [reply src/components/RegionCard/styles.ts:5]: Over the export?

Comment (eirsture) [reply src/components/RegionCard/styles.ts:5]: Just within the regionCard styling somewhere :)

File: src/styles/card.ts [hunk 3]
```diff
@@ -59,11 +59,15 @@ export const cardImg = style({
 });
 
 export const secondaryCard = style({
+    maxWidth: '400px',
```
Comment (eirsture) [inline src/styles/card.ts:62]: If you want to remove the underlines when hovering on desktop you can add: `textDecoration: 'none',`

**Review by eirsture — APPROVED** [review]

File: src/styles/card.ts (hunk 3 above)
Comment (jennyjyu) [reply src/styles/card.ts:62]: funka ikke hos mæ....

Comment (eirsture) [reply src/styles/card.ts:62]: åja, hmm. Bare dropp det da

Comment (jennyjyu) [reply src/styles/card.ts:62]: Gets included in another branch!

**Pull request merged**